src/capitalsFirst.js

```diff
@@ -18,7 +18,20 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  let UpperStrArr = [];
+  let LowerStrArr = [];
+  let arrWords = str.split(' ');
+  for (let i = 0; i < arrWords.length; i++) {
+    if (arrWords[i][0].toLowerCase() === arrWords[i][0].toUpperCase()) {
```

Comment:
this is working but it's not obvious what is happening when you read the code. 

I'd recommend comparing arrwords[i][0] with 'A', 'Z', 'a' and 'z'. You can compare chars with '>', '<'.

At least you need to add a comment what is happening here.

Reply:
after thinking more about this I like your solution better than what I've proposed. Accepted. Please remove the comment. Respond here to discuss reviewers feedback.